src/js/footer/050-furnace-code.js

```diff
@@ -64,3 +63,13 @@ function ToggleNPM( $this ) {
 function ClearNPM() {
 	furnaceNPM.innerHTML = 'npm install --save @gov.au/core';
 }
+
+/**
+ * Select all button selects all checkbox elements on page
+ */ 
+AddEvent( selectAll, 'click', function( event ){
+	furnaceComponents.forEach(function(furnaceComponents) {
```

Comment:
I would probably rename the callback argument `furnaceComponents` to something a bit less confusing like `element` or `item`. Having the array the same name as the iterated item reads a bit funny